packages/dev/core/src/Rendering/prePassRenderer.ts

```diff
@@ -155,7 +155,7 @@ export class PrePassRenderer {
         {
             purpose: Constants.PREPASS_DEPTH_TEXTURE_TYPE,
             type: Constants.TEXTURETYPE_FLOAT,
-            format: Constants.TEXTUREFORMAT_R,
+            format: Constants.TEXTUREFORMAT_RGBA,

```

Comment:
seems to be a breaking change

Reply:
This was a mistake. I set a bunch of textures to RGBA so that they render in Spector.js.